### ISSUE TITLE
Implement VIM bindings mode for text input

### ISSUE THREAD

**Issue by evendaniel** [a85/POSTMan-Chrome-Extension#12820]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

No

### Describe the solution you'd like

I'd like the option to use standard VIM keyboard bindings and modal editing with Postman for writing tests/configurations/values/etc. Anything with a text field I would like to be able to use the keybindings I am familiar with while working with Postman.

**Comment by monoira** [comment]
I agree. I love vim :+1:  :smile:

**Comment by WilliamWelsh** [comment]
please lord god in all things that are holy let us have thy blessed vim mode bestowed upon us in this god forsaken electron app so we may be able to move swiftly in our raw window. blessed are those that implement this. blessed are those that have been enlightened by thy motions.

**Comment by opxyc** [comment]
+1 :heart:

**Comment by crazyoptimist** [comment]
+1 here

**Comment by rrojan** [comment]
+1000 here

**Comment by eliseucbrito** [comment]
+1!!!!

**Comment by dfenerski** [comment]
sorry for the \<\<bump>>

**Comment by ghost** [comment]
+1 here

**Comment by TheGreatestAlan** [comment]
+1

**Comment by Arnav-Bista** [comment]
+1

**Comment by dirtycache** [comment]
Yep. +1. Please. Tried the VS Code extension for specifically this reason.  Quite dismayed to discover it ignores my VS Code global use of vim keys.

**Comment by timmyjose** [comment]
+1

**Comment by BennisonDevadoss** [comment]
+1

**Comment by mrwizard82d1** [comment]
Agree. +1.

**Comment by callen-rti** [comment]
+100

**Comment by vysogot** [comment]
+n+1

**Comment by khanhbq2k** [comment]
+n+2

**Comment by Niwau** [comment]
+1

**Comment by 3ther-joyboy** [comment]
+1

**Comment by 2uan2** [comment]
+n

**Comment by mrehanabbasi** [comment]
More than an year now since this was requested and all I see are `+x`. Can someone please add it already?

**Comment by n3tw0rth** [comment]
that's great if we can add that

**Comment by luis-duque-azumo** [comment]
+1

**Comment by aaborobbie** [comment]
+1

**Comment by yamanidev** [comment]
+1

I was certain that if I didn't find the feature itself, I'd find loads of people requesting it xD. Once we go vim, editing text without it becomes so foreign, and I love that.

**Comment by AbsoluteZero000** [comment]
+1

**Comment by gznrf** [comment]
+10!

**Comment by claudioge** [comment]
+1

**Comment by Gabriel-Spinola** [comment]
+1

**Comment by pawel-mackiewicz** [comment]
bump +1

**Comment by orkansama** [comment]
bump +1

**Comment by katripathi-fadv** [comment]
bump +

**Comment by distantcompany** [comment]
bump +1

**Comment by mlsquires** [comment]
just about every development tool I use supports this. A PITA to start editing in Postman and remember that I can't use vi bindings.

**Comment by diego-lutke** [comment]
+1
